storybook/cypress/e2e/accordion.cy.ts

```diff
@@ -370,4 +371,21 @@ describe("Accordion", () => {
       .its("scrollY")
       .should("equal", 0);
   });
+
+  it("emit dsoToggleSectionEnd event for all toggle animations end", () => {
+    cy.get("dso-accordion")
+      .wait(1000)
+      .find("dso-accordion-section")
+      .first()
+      .shadow()
+      .find(".dso-section-handle")
+      .realClick()
+      .wait(1000)
+      .get("@dsoToggleSectionAnimationEndListener")
+      .should("have.been.calledTwice")
+      .invoke("getCalls")
+      .invoke("at", -1)
+      .its("args.0.detail.section.open")
```

Comment:
Dit voelt als iets wat incidenteel fout kan gaan. Ze eindigen als het goed is 'tegelijk'. weten we echt 100% zeker dat de laatste altijd degene is die opent?

Reply:
Dit heeft nog wat aandacht nodig.

Reply:
Opgelost door allowMultiple toe te staan, zodat bij deze test er maar 1 event emit.